src/Plugin/Field/FieldWidget/EntityReference.php

```diff
@@ -209,8 +210,14 @@ function formElement(FieldItemListInterface $items, $delta, array $element, arra
     );
 
     $ids = [];
-    if (($trigger = $form_state->getTriggeringElement()) && !empty($trigger['#ajax']['event']) && $trigger['#ajax']['event'] == 'entity_browser_value_updated' && in_array($this->fieldDefinition->getName(), $trigger['#parents'])) {
-      if ($value = $form_state->getValue($trigger['#parents'])) {
+    if (($trigger = $form_state->getTriggeringElement()) && in_array($this->fieldDefinition->getName(), $trigger['#parents'])) {
```

Comment:
We could add a comment here, because it's not immediately clear why parents for this element aren't always the same.